ebos/eclfluxmodule.hh

```diff
@@ -271,7 +272,10 @@ protected:
 
             const Evaluation& pressureInterior = intQuantsIn.fluidState().pressure(phaseIdx);
             Evaluation pressureExterior = Toolbox::value(intQuantsEx.fluidState().pressure(phaseIdx));
-            pressureExterior += rhoAvg*(distZ*g);
+            if (enableExtbo) //z-dependency ...
+                pressureExterior += Toolbox::value(rhoAvg)*(distZ*g);
+            else
```

Comment:
Maybe an explanation why this is needed?

Reply:
Ok